generator.rb

```diff
@@ -0,0 +1,72 @@
+def is_vowel?(letter)
+    result = false
+    if (letter == "a"|| letter == "e" || letter == "i" || letter == "o" || letter == "u")
+        result = true
+    end
+    return result
+end
+
+    #test1 = "i"
+    #puts "Is #{test1} a vowel?"
+    #puts is_vowel?(test1)
+
+    #test2= "v"
+    #puts "Is #{test2} a vowel?"
+    #puts is_vowel?(test2)
```

Comment:
If you have debugging code, please remove it before you submit! This is a good habit to get into for future jobs too!